chain/client/src/client.rs

```diff
@@ -133,6 +134,7 @@ pub struct Client {
     pub block_sync: BlockSync,
     /// Keeps track of syncing state.
     pub state_sync: StateSync,
+    state_parts_arbiter_handle: ArbiterHandle,
```

Comment:
Why do you need `Client` to have an `ArbiterHandle`?
The previous approach seems cleaner to me, as it keeps contstructors of `Client` and `ClientActor` smaller.

Previous approach: `state_parts_arbiter` is an internal detail of `ClientActor`. `ClientActor` passes it to `Client` which passes it to `StateSync::run()` when needed.

Reply:
Makes sense. I will only keep it in `ClientActor` and pass it to the `StateSync::run()`